index.html

```diff
@@ -127,13 +130,13 @@ <h2>Project name goes here</h2>
       <div class="proj-1">
         <img src="assets/Proj5.png" alt="project 5" />
         <div class="description">
-          <h2>Project name goes here</h2>
-          <ul>
-            <li>HTML/CSS</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
+          <h2 class="title">Project name goes here</h2>
+          <ul class="stacks">
+            <li class="stacks-1">HTML/CSS</li>
+            <li class="stacks-2">Ruby on Rails</li>
+            <li class="stacks-3">JavaScript</li>
```

Comment:
As part of the project requirements, each project card should be rendered dynamically with JavaScript and as such the projects container shouldn't hold any html. Consider doing so in order to meet the requirement and to have a thorough experience rendering data dynamically.

Reply:
- [ ] As indicated by the previous reviewer, as part of the project requirements, each project card should be rendered dynamically with JavaScript and as such the projects container shouldn't hold any html. Consider doing so in order to meet the requirement and to have a thorough experience rendering data dynamically.